avocado/core/varianter.py

```diff
@@ -416,11 +416,11 @@ def is_parsed(self):
         """
         return self.variants is not None
 
-    def _skip_new_data_check(self, fction, args):
+    def _skip_new_data_check(self, function, args):
```

Comment:
Well, OK, function is not a builtin. Still I don't like this much but I'm biased because of my bash background... Anyway I'm fine with this change if you really want it.